src/main/features/core/lastFM.js

```diff
@@ -1,9 +1,30 @@
 import { BrowserWindow } from 'electron';
-import { LastFmNode } from 'lastfm';
 import path from 'path';
 
 import { LASTFM_API_KEY, LASTFM_API_SECRET } from '../../constants';
 
+// hijack lastfm module to use 'electron.net' instead of 'http'
+global.GENTLY_HIJACK = {
+  hijack(require) {
+    return function (moduleName) {
```

Comment:
I'll change it for an arrow function since it seems that's the norm on the code (I didn't realize earlier 🙇)